### PR TITLE
Fixes #181 - Add an Infrastructure section

### LINKED ISSUE #181: Add Infrastructure section
We need to link to the infra standard, and we can link to others (possibly moving the fetch integration section to there, https://wicg.github.io/ua-client-hints/#fetch-integration). 

We can also explicitly point to GREASE and client hints infra.

### PR DESCRIPTION
**Pull request by miketaylr**
And move the "integration with fetch" section into it.

PTAL @amtunlimited

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/ua-client-hints/pull/195.html" title="Last updated on Feb 9, 2021, 3:24 PM UTC (84573f4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/ua-client-hints/195/2b830e6...miketaylr:84573f4.html" title="Last updated on Feb 9, 2021, 3:24 PM UTC (84573f4)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -177,6 +177,22 @@ the underlying platform in the next request:
...
+Standard.
+[[!CLIENT-HINTS-INFRASTRUCTURE]]
+[[!I-D.ietf-httpbis-client-hints]]
+[[!INFRA]]
+
+Some of the terms used in this specification are defined in <cite>Structured Field Values for
+HTTP</cite>.
+[[!I-D.ietf-httpbis-header-structure]]
+
+[=Client Hints Fetch integration=] is defined as part of the [[CLIENT-HINTS-INFRASTRUCTURE]]
```
Comment (amtunlimited) [inline index.bs:193]: I think if you site the infra document you don't need the specific Fetch bit

(It's also a little weird to bring up the fetch and not the HTML, but that's whatever)

**Review by amtunlimited — COMMENTED** [review]
LGTM

File: index.bs (hunk 1 above)
Comment (miketaylr) [reply index.bs:193]: Thanks, that's fair.

Can you clarify what you mean by:

> (It's also a little weird to bring up the fetch and not the HTML, but that's whatever)

Are you saying it's weird that there isn't a reference to https://wicg.github.io/client-hints-infrastructure/#html? That's a fair point, I was just moving the existing section 2.8 (https://wicg.github.io/ua-client-hints/#fetch-integration).

Comment (amtunlimited) [reply index.bs:193]: It was just a comment. Either way, I don't think we need it if just bring up the infra spec as a whole

> (edit: I will never use the word fair again, promise)

Also fair :P

**Review by amtunlimited — APPROVED** [review]

**Pull request merged**